frontend/src/scenes/billing/billingLogic.tsx

```diff
@@ -234,11 +234,18 @@ export const billingLogic = kea<billingLogicType>([
                     // for customers running into performance issues until we have a more permanent fix
                     // of splitting the billing and forecasting data.
                     const skipForecasting = values.featureFlags[FEATURE_FLAGS.BILLING_SKIP_FORECASTING]
-                    const response = await api.get(
-                        'api/billing' + (skipForecasting ? '?include_forecasting=false' : '')
-                    )
+                    try {
+                        const response = await api.get(
+                            'api/billing' + (skipForecasting ? '?include_forecasting=false' : '')
+                        )
 
-                    return parseBillingResponse(response)
+                        return parseBillingResponse(response)
+                    } catch (error: any) {
+                        if (error.data?.code === 'no_license') {
+                            return null
+                        }
+                        throw error
+                    }
```

Comment:
Just out of curiosity - what was the reason for making these changes to license error and how we handle it in billing logic etc? I realize it's handling it more gracefully but curious about reason and downstream impact previous version was causing.